src/index.html

```diff
@@ -22,6 +22,408 @@
   </head>
 
   <body>
-    <h1>Catalog</h1>
+    <header class="header">
+      <a
+        href="#logo"
+        class="logo_container"
+      >
+        <img
+          src="/src/images/logo.png"
+          alt="moyo"
+          class="logo"
+        />
+      </a>
+      <nav class="nav">
+        <ul class="nav__list">
+          <li class="nav__item">
+            <a
+              href="#Apple"
+              class="nav__link is-active"
+            >
+              Apple
+            </a>
+          </li>
+
+          <li class="nav__item">
+            <a
+              href="#Samsung"
+              id="Samsung"
+              class="nav__link"
+            >
+              Samsung
+            </a>
+          </li>
+
+          <li class="nav__item">
+            <a
+              href="#Smartphones"
+              id="Smartphones"
+              class="nav__link"
+            >
+              Smartphones
+            </a>
+          </li>
+
+          <li class="nav__item">
+            <a
+              href="#Laptops&Computers"
+              id="Laptops&Computers"
+              class="nav__link"
+              data-qa="nav-hover"
+            >
+              Laptops & Computers
+            </a>
+          </li>
+
+          <li class="nav__item">
+            <a
+              href="#Gadgets"
+              id="Gadgets"
+              class="nav__link"
+            >
+              Gadgets
+            </a>
+          </li>
+
+          <li class="nav__item">
+            <a
+              href="#Tablets"
+              id="Tablets"
+              class="nav__link"
+            >
+              Tablets
+            </a>
+          </li>
+
+          <li class="nav__item">
+            <a
+              href="#Photo"
+              id="Photo"
+              class="nav__link"
+            >
+              Photo
+            </a>
+          </li>
+
+          <li class="nav__item">
+            <a
+              href="#Video"
+              id="Video"
+              class="nav__link"
+            >
+              Video
+            </a>
+          </li>
+        </ul>
+      </nav>
+    </header>
+
+    <main class="cards-container">
+      <form
```

Comment:
This isn't form, use the semantic tag for card, fix it everywhere
```suggestion
      <article
```